manifests/plugin/write_http.pp

```diff
@@ -1,7 +1,8 @@
 # https://collectd.org/wiki/index.php/Plugin:Write_HTTP
 class collectd::plugin::write_http (
-  $ensure    = 'present',
-  Hash $urls = {},
+  $ensure     = 'present',
+  Hash $nodes = {},
+  Hash $urls  = {}
```

Comment:
can you enforce the structure of the hash? Examples are the last three here: https://puppet.com/docs/puppet/5.4/lang_data_hash.html#examples

Reply:
Added Hash structure definition